src/index.html

```diff
@@ -11,9 +11,29 @@
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Document</title>
   <link rel="stylesheet" href="./style.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
 </head>
 <body>
-
+<div class="card" data-qa="card">
+  <img src="images/imac.jpeg" alt="imac" class="product">
+  <div class="wrap">
+    <p class="article">Моноблок APPLE A1419 iMac 2727" Retina 5K (MNED2UA/A)</p>
+    <p class="code">Код товара: 195434</p>
+    <div class="review">
+      <img src="images/star-active.svg" alt="star-active">
+      <img src="images/star-active.svg" alt="star-active">
+      <img src="images/star-active.svg" alt="star-active">
+      <img src="images/star-active.svg" alt="star-active">
+      <img src="images/star.svg" alt="star">
+      <p>Отзывов: 5</p>
+    </div>
+    <div class="price">
+      <span class="left">Цена:</span>
+      <span class="right">69 999 грн</span>
+    </div>
+    <button data-qa="hover">Купить</button>
```

Comment:
In the task description, there is a point that it should be a link. Please read the description carefully before starting.
![image](https://user-images.githubusercontent.com/29413892/56793841-8e267e00-6815-11e9-934d-5c0c54146550.png)

Reply:
By the way, `button` takes System UI font. So you have to overwrite it explicitly. Now your button has this font (the Apple ones)
![image](https://user-images.githubusercontent.com/29413892/56795244-b9f73300-6818-11e9-895c-ec6d8eac2872.png)